Common Word Definitions.md

```diff
@@ -0,0 +1,33 @@
+Common Word Definitions
```

Comment:
It's a side thing for right now,  I was going to be putting in definitions, and also a link to those words in just the apprentice badge.